torch/csrc/inductor/aoti_torch/c/shim.h

```diff
@@ -54,6 +54,10 @@
 // The following files are implemented in a header-only way and are guarded by
 // test/cpp/aoti_abi_check
 #include <c10/util/BFloat16.h>
+#include <c10/util/Float8_e4m3fn.h>
+#include <c10/util/Float8_e4m3fnuz.h>
+#include <c10/util/Float8_e5m2.h>
+#include <c10/util/Float8_e5m2fnuz.h>
```

Comment:
Why include all 4 flavors, but define functions for only 2? Also, why not use forward declaration from lines 686-687